dev-python/pytest-randomly/pytest-randomly-3.12.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Pytest plugin to randomly order tests and control random.seed"
+HOMEPAGE="
+	https://github.com/pytest-dev/pytest-randomly/
+	https://pypi.org/project/pytest-randomly/
+"
+SRC_URI="$(pypi_sdist_url --no-normalize)"
+
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	test? (
+		dev-python/pytest-xdist[${PYTHON_USEDEP}]
+		dev-python/factory-boy[${PYTHON_USEDEP}]
+		dev-python/Faker[${PYTHON_USEDEP}]
+		$(python_gen_cond_dep 'dev-python/numpy[${PYTHON_USEDEP}]' python3_{9,10})
+	)
+"
+
+RDEPEND="
+	dev-python/pytest[${PYTHON_USEDEP}]
+	$(python_gen_cond_dep '>=dev-python/importlib_metadata-3.6.0[${PYTHON_USEDEP}]' python3_9)
```

Comment:
Use `3.9` instead of `python3_9`, that matches all implementations that have 3.9 stdlib.